docs/services/gcp-services-guide.md

```diff
@@ -59,40 +57,19 @@ export SUBNETWORK_NAME=default # the subnetwork that your GKE cluster lives in.
 ### GKE Cluster Steps
 
 We are assuming you've got a GKE cluster up and running created either via
-GCP Console or gcloud CLI. If you don't, the basic command to create one is
-the following:
+GCP Console or gcloud CLI.
 
-1. Create a GKE cluster:
-```bash
-gcloud beta container --project "${PROJECT_ID}" clusters create "my-existing-cluster" \
-  --zone "us-central1-a" \
-  --cluster-version "1.13.7-gke.8" \
-  --machine-type "n1-standard-1" \
-  --image-type "COS" \
-  --disk-type "pd-standard" \
-  --disk-size "100" \
-  --metadata disable-legacy-endpoints=true \
-  --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" \
-  --num-nodes "1" \
-  --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --enable-autoupgrade --enable-autorepair --no-enable-basic-auth \
-  --network "projects/${PROJECT_ID}/global/networks/${NETWORK_NAME}" \
-  --subnetwork "projects/${PROJECT_ID}/regions/us-central1/subnetworks/${SUBNETWORK_NAME}" \
-  --default-max-pods-per-node "110" \
-  --addons HorizontalPodAutoscaling,HttpLoadBalancing
-```
-
-This may take a while to finish.
-
-2. Connect to your GKE cluster
+1. Connect to your GKE cluster
 ```bash
 gcloud container clusters get-credentials "my-existing-cluster" --zone us-central1-a --project "${PROJECT_ID}"
 ```
 
-3. Make sure `kubectl` is able to communicate with your GKE cluster
+2. Make sure `kubectl` is able to communicate with your GKE cluster
 ```bash
 kubectl cluster-info
 ```
-4. Make sure Helm is installed with permissions to work on `crossplane-system`
+
+3. Make sure Helm is installed with permissions to work on `crossplane-system`
```

Comment:
I feel like we could probably start the steps here - i.e. link off to some GKE docs and assume the user can figure out how to get GKE running and connect to it using `kubectl`.

Reply:
Generally agree that shorter is better, though the short verification steps help orient and make sure they're ready for the next step. Would be fine either way IMO.